mongo/mongo.go

```diff
@@ -5,47 +5,77 @@ import (
 	"errors"
 	"strconv"
 
-	"github.com/globalsign/mgo"
-
-	"github.com/globalsign/mgo/bson"
-
+	"github.com/ONSdigital/dp-healthcheck/healthcheck"
+	dpMongoHealth "github.com/ONSdigital/dp-mongodb/v2/health"
+	dpMongoDriver "github.com/ONSdigital/dp-mongodb/v2/mongodb"
 	errs "github.com/ONSdigital/dp-recipe-api/apierrors"
 	"github.com/ONSdigital/dp-recipe-api/models"
 	"github.com/ONSdigital/log.go/log"
+	"go.mongodb.org/mongo-driver/bson"
+)
+
+const (
+	connectTimeoutInSeconds = 5
+	queryTimeoutInSeconds   = 15
```

Comment:
These feel like the should be config driven - we might need to tweak these on the fly and thats easier if its a config update rather than a new release.